### DISCUSSION TITLE
Sending an onion message

### DISCUSSION THREAD

**Discussion by valentinewallace** [ACINQ/eclair#2205] [Developers]
Hi! I'm using Polar to try to send an onion message between Eclair nodes, with an LDK node in the middle. However, I'm getting an error.

I'm copying the Eclair API docs example almost exactly so wondering what I'm doing wrong here:

```
> eclair-cli signmessage --content=2b03ffffff --recipientNode=033ce23ae96c271e4112ce69acd5e0fc69b9287ed5ed62e39720841ed06753ef9b --intermediateNodes=030b7f70f142df26df7104a3c5e0aaaa91c65f7ace785fe180aadc2951b59f503d 

Request is missing required form field 'msg'
```

Thanks in advance :)

**Comment by pm47** [comment]
Hello! There is typo in our doc, it's actually  `sendonionmessage` instead of `signmessage`

**Reply by valentinewallace** [reply]
Ah, makes sense. TY
